doc/arrays.md

```diff
@@ -17,6 +17,14 @@ Sniff provides the following settings:
 * `spacesAroundBrackets`: number of spaces you require to have around array brackets
 * `enableEmptyArrayCheck` (defaults to `false`): enables check for empty arrays
 
+#### SlevomatCodingStandard.Arrays.Sorted 🔧
```

Comment:
We already have `AlphabeticallySortedUses` so the name can be `AlphabeticallySortedKeys`